src/gui/transferlistsortmodel.cpp

```diff
@@ -90,6 +85,17 @@ namespace
         return isLeftValid ? -1 : 1;
     }
 
+    int compareAsBool(const QVariant &left, const QVariant &right)
+    {
+        const bool leftValid = left.isValid();
+        const bool rightValid = right.isValid();
+        if (leftValid && rightValid)
+            return threeWayCompare(left.toBool(), right.toBool());
+        if (!leftValid && !rightValid)
+            return 0;
+        return rightValid ? -1 : 1;
```

Comment:
Just double checking. Usually it is `return leftValid ? -1 : 1;` and this is special behavior, right?

Reply:
> and here is special behavior, right?

Done by mistake. Fixed.